2023-risks/m2-insecure-communication.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: col-sidebar
+title: "M2: Insecure Communication"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Most modern mobile applications exchange data with one or more remote servers. When the data transmission takes place, it typically goes through the mobile device's carrier network and the internet, a threat agent listening on the wire can intercept and modify the data if it transmitted in plaintext or using a deprecated encryption protocol. The following threat agents exist:
+
+- An adversary that shares your local network (compromised or monitored Wi-Fi);
+- Rogue carrier or network devices (routers, cell towers, proxy's, etc); or
+- Malware on your mobile device.
+
+# Attack Vectors	
+
+**Exploitability EASY**
+
+While modern applications do reply on cryptographic protocols such as SSL/TLS, they can sometimes have flaws in their implementations like: 
+
+- No hostname validation;
+- Accepting bad ssl certificates (self-signed, revoked, expired..); or
+- Inconsistency (having SSL/TLS only on select workflows such as authentication).
+
+
+# Security Weakness	
+
+**Prevalence COMMON** 
+
+**Detectability AVERAGE**
+
+while modern mobile applications protect network traffic, they often leave some requests unprotected. This inconsistency leads to the risk of exposing data and session IDs to interception. The use of transport security does not mean the app has implemented it correctly. To detect basic flaws, observe the phone's network traffic. More subtle flaws require inspecting the design of the application and the applications configuration.  
+
+# Technical Impacts	
+
+**Impact SEVERE**
+
+This flaw can expose user data which might lead to account takeover, user impersonation, PII data leaks and more, for instance an attacker might intercept user credentials, session, 2FA tokens which can open the door for more elaborate attacks.   
+
+# Business Impacts
+	
+**Application / Business Specific** 
+
+At a minimum, interception of sensitive data through a communication channel will result in a privacy violation.
+
+The violation of a user's confidentiality may result in:
+
+- Identity theft;
+- Fraud, or
+- Reputational Damage.
+
+# Am I Vulnerable To 'Insecure Communication'?
```

Comment:
I would expect the advice to try to "penetration test" the connection, e.g, with a packet inspector or trying to MITM